spring-cloud-gcp-data-spanner/src/test/java/org/springframework/cloud/gcp/data/spanner/test/IntegrationTestConfiguration.java

```diff
@@ -104,6 +105,7 @@ public TradeRepositoryTransactionalService tradeRepositoryTransactionalService()
 	}
 
 	@Bean
+	@ConditionalOnMissingBean
```

Comment:
Why not sure remove these two bean definitions from here?

Reply:
good point

Reply:
We don't use this configuration file. I removed these beans from the emulator's test spring configuration.